mtp_bank_admin/apps/bank_admin/refund.py

```diff
@@ -0,0 +1,38 @@
+import csv
+import io
+from datetime import datetime
+
+from moj_auth.backends import api_client
+
+from .exceptions import EmptyFileError
+
+OUTPUT_FILENAME = 'mtp_accesspay_%s.csv'
+
+
+def generate_refund_file(request):
+    client = api_client.get_connection(request)
+    refund_transactions = client.bank_admin.transactions.get(status='refund_pending')
+
+    with io.StringIO() as out:
+        writer = csv.writer(out)
+
+        refunded_transactions = []
+        for transaction in refund_transactions:
+            writer.writerow([
+                transaction['sender_sort_code'],
+                transaction['sender_account_number'],
+                transaction['sender_name'],
+                transaction['amount'],
+                transaction['reference']
```

Comment:
Oh yes. @ibrechin all the records in the AccessPay will have the same reference, this is what people see in their bank statement.

I don't remember the exact sentence, something like: "Payment refund" should work for now, you could set it as a constant (in `settings.py` maybe) so that we can easily change it in the future.

Does it make sense?